.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+name: build
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  builds:
+    name: Earliest Supported Version
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        ocaml-version:
+          - 4.04.0
```

Comment:
Did you go with 4.04 as the earliest supported version on purpose? We only test from 4.08.01 and up.